### PR TITLE
LPS-116020 Fix indexType property

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java [hunk 1]
```diff
@@ -391,6 +391,12 @@ else if (StringUtil.equals(type, "textarea")) {
 					_upgradeTextArea(companyId, jsonObject);
 				}
 
+				if (!StringUtil.equals(type, "separator") &&
+					Validator.isNull(jsonObject.getString("indexType"))) {
```
Comment (leoadb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java:395]: @marcelabc We can replace **Validator.isNull** with **jsonObject.isNull**. We can also move this operand to be the first one. Thanks!

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**